control/trajectory_follower_node/param/test.param.yaml

```diff
@@ -0,0 +1,12 @@
+/**:
+  ros__parameters:
+    lateral_controller_mode: "mpc"
```

Comment:
Since these params are only used for the test, would you move these under `test` directory? 
https://github.com/autowarefoundation/autoware.universe/tree/9983ead76b3fc164f23aab6d8fe1d27519f1770c/control/trajectory_follower_node/test

Or, ideally, we should test for all options (not only for the `mpc` but also `pure_pursuit`), so I guess these params should not be in the param file, but could be defined in the test directly.

Reply:
Check my latest comment, whether it resolves your issue.

Reply:
thank you!